main/EDPftvarcon.F90

```diff
@@ -167,8 +171,8 @@ real(r8), allocatable :: allom_smode(:)        ! sapwood allometry function type
      ! Prescribed Physiology Mode Parameters
      real(r8), allocatable :: prescribed_npp_canopy(:)           ! this is only for the special 
                                                                  ! prescribed_physiology_mode
-     real(r8), allocatable :: prescribed_npp_understory(:)       ! this is only for the special 
```

Comment:
Do we need 'special' in all of these notes? :)

Reply:
I'll defer to @ckoven on this one. I don't think I meant to change any of the comments here.

Reply:
i guess all the modes are special in their own way, happy to let the word get deleted here.